src/plugins/intel_cpu/src/nodes/normalize.cpp

```diff
@@ -1483,7 +1483,8 @@ std::shared_ptr<NormalizeL2::NormalizeL2Executor> NormalizeL2::NormalizeL2Execut
               OV_CASE2(Precision::U8, Precision::FP32, uint8_t, float),
               OV_CASE2(Precision::I8, Precision::FP32, int8_t, float),
               OV_CASE2(Precision::FP32, Precision::FP32, float, float),
-              OV_CASE2(Precision::BF16, Precision::BF16, bfloat16_t, bfloat16_t));
+              OV_CASE2(Precision::BF16, Precision::BF16, bfloat16_t, bfloat16_t),
+              OV_CASE2(Precision::FP16, Precision::FP16, float16_t, float16_t));
```

Comment:
That instance is not valid for NormalizeL2JitExecutor since it doesn't support FP16 precision. 
So we need to fallback on FP32 precision for x64 arch insdie initSupportedPrimitiveDescriptors

Reply:
@dmitry-gorokhov corrected